docs/networking/best-practice/overview.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 51
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Network
+  - network
+  - VLAN
+  - vlan
+Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
+---
+
+# Overview
+
+In best practice setcion, we always recommend your machine with multiple NICs for node access and VM networks. Please refer the multiple NICs best practice if your machines have multiple NICs. Otherwise, refer single NIC best practice. 
```

Comment:
```suggestion
In the [best practice](#best-practice) section below, we always recommend to configure multiple NICs in your machine for node access and VM networks. Please refer the [multiple NICs](multiple-nics-vlan-aware-switch.md) best practice if your machines have multiple NICs. Otherwise, please refer to the [single NIC](single-nic-vlan-aware-switch.md) best practice. 
```